code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -74,10 +76,48 @@
 	return ..()
 
 /mob/living/simple_animal/hostile/abnormality/mountain/Move()
+	var/turf/floor = get_turf(src)
+	for(var/thing in floor)
```

Comment:
Would probably be a smart idea to iterate through var/obj/thing instead.

Reply:
Good idea